pnpm-lock.yaml

```diff
@@ -943,6 +943,10 @@ packages:
       '@types/node': 16.11.11
     dev: true
 
+  /@types/unist/2.0.6:
```

Comment:
I wouldn't expect new packages to be added since no `package.json` was updated

Reply:
Correct, I've removed all changes from pnpm-lock.yaml